### ISSUE TITLE
Aria-label rendering seems to be chained with useHTML

### ISSUE THREAD

**Issue by LeoJiang1993** [highcharts/highcharts#23164]
#### Expected behaviour
Aria-Label should be rendered like "1. <7, 5 cases."

#### Actual behaviour
It is escaping html tag, result is "1. ".

#### Product version

highcharts: 9.3.3
highcharts-react-official: ^3.0.0

#### Affected browser(s)
Not seems to be browser specific.

**Comment by KacperMadej** [comment]
Hi @LeoJiang1993 

Thank you for reporting about the problem.

Please add a minimal chart configuration and setup needed to be used to recreate the issue you are having.
It works fine for:
```js
    lang: {
        accessibility: {
            exporting: {
                menuButtonLabel: '1. <7, 5 cases.'
            }
        }
    },
```
Demo: https://jsfiddle.net/BlackLabel/6urojhkn/

![Image](https://github.com/user-attachments/assets/33f3b3bf-5255-46cc-9c44-7e4483ce69b4)

**Comment by KacperMadej** [comment]
Unable to recreate. Closing due to inactivity. If you have more info (like the requested steps that should be used to recreate the problem) please share and the issue could be reopened.
